src/chart/helper/createGraphFromNodeEdge.js

```diff
@@ -39,10 +40,15 @@ export default function (nodes, edges, seriesModel, directed, beforeLink) {
     var linkNameList = [];
     var validEdges = [];
     var linkCount = 0;
+
+    // auto curveness
+    calculateMutilEdges(edges, seriesModel, graph);
```

Comment:
I think the calling of `calculateMutilEdges` is probably not proper to be here. The reasons are:
+ `createGraphFromNodeEdge.js` is not only be used by `graph series`, but also used by `sankey series`.
+ we should better not use the raw `edges` as the input of `calculateMutilEdges`. The reason is described below.

If we can ensured that the algorithm only depends on the `data`, I think it can be put at [the end of `getInitialData` of `GraphSeries.js`](https://github.com/apache/incubator-echarts/blob/4.8.0/src/chart/graph/GraphSeries.js#L113).
And use `graph` (that is, the return of `createGraphFromNodeEdge`) as the input instead of the raw user input `edges`. Edges info can be retrieved from the date structure `graph`.

And each time of calling `calculateMutilEdges`, the previous calculation result mounted on "graph series instance" (that is, `linkMap` and `curvenessList`) should be discarded.